templates/tools/dockerfile/clang_update.include

```diff
@@ -0,0 +1,32 @@
+#=================
+# Update clang to a version with improved tsan
+
+RUN apt-get update && apt-get -y install python cmake && apt-get clean
+
+RUN git clone -n -b release_38 http://llvm.org/git/llvm.git && ${'\\'}
```

Comment:
So... that's a lot of hash.

Can I request a script (tools/im_not_sure_where/update_clang_version.sh) that takes a branch name and outputs the current hashes?